packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -4191,7 +4191,7 @@ function doubleInvokeEffectsOnFiber(
     }
     reappearLayoutEffects(root, fiber.alternate, fiber, false);
     if (shouldDoubleInvokePassiveEffects) {
-      reconnectPassiveEffects(root, fiber, NoLanes, null, false);
+      reconnectPassiveEffects(root, fiber, NoLanes, null, false, 0);
```

Comment:
It's not a new issue in this PR but an existing issue is that all these phases end up double logging the effects.